### PR TITLE
ui: translations

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

**Comment by CIAvash** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by xe1st** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by xe1st** [comment]
recheck

**Comment by summoner001** [comment]
I have read the CLA Document and I hereby sign the CLA

File: apps/multiplatform/common/src/commonMain/resources/MR/tr/strings.xml [hunk 1]
```diff
@@ -1289,7 +1289,7 @@
     <string name="stop_snd_file__message">Dosya gönderimi durdurulacaktır.</string>
     <string name="moderated_description">yönetildi</string>
     <string name="error_creating_member_contact">Üye bağlantısı oluşturulurken hata</string>
-    <string name="sending_delivery_receipts_will_be_enabled">Tüm kişiler için iletim bilgisi gönderme özelliği etkinleştirilecek</string>
+    <string name="sending_delivery_receipts_will_be_enabled">[Suggested by translate] Tüm kişiler için teslimat makbuzlarının gönderilmesi etkinleştirilecektir.</string>
```
Comment (epoberezkin) [inline apps/multiplatform/common/src/commonMain/resources/MR/tr/strings.xml:1292]:
```suggestion
    <string name="sending_delivery_receipts_will_be_enabled">Tüm kişiler için iletim bilgisi gönderme özelliği etkinleştirilecek</string>
```

**Pull request merged**